__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/development-data/index')
+
+
+beforeEach(()=> seed(testData))
+afterAll(()=>db.end())
+
+describe('GET /api/topics', () => {
+  test('responds with an array of topic objects', () => {
+      return request(app)
+          .get('/api/topics')
+          .expect(200)
+          .then(({ body }) => {
+              expect(body.topics).toBeInstanceOf(Array);
+              body.topics.forEach((topic) => {
+                  expect(topic).toEqual(
+                      expect.objectContaining({
+                          slug: expect.any(String),
+                          description: expect.any(String),
+                      })
+                  );
+              });
+          });
+  });
+
+  test('responds with a 404 for an invalid path', () => {
+      return request(app)
+          .get('/api/nonexistent')
+          .expect(404)
+          .then(({ body }) => {
+              expect(body.msg).toBe('Path not found');
+          });
+  });
```

Comment:
Great cover of invalid endpoints. As it's for dealing with all the invalid endpoints, so it's better be outside of the describe of ('/api/topics'). Also it'll be good to wrap all the endpoints into one big describe